Unit.h

```diff
@@ -20,29 +20,11 @@
 #include <iostream>
 #include <fstream>
 #include <vector>
+#include <cmath>
```

Comment:
Ezt elég csak a cpp-ben include-olni, itt nincs használva, ha jól látom.

Reply:
ez, és minden más include is át lett helyezve